backend/gmailAuth.js

```diff
@@ -25,28 +29,16 @@ module.exports.authSend = function authSend(receivers, subject, message) {
  * @param {Object} credentials The authorization client credentials.
  * @param {function} callback The callback to call with the authorized client.
  */
-function authorize(receivers, subject, message, callback) {
-  const CLIENT_ID = process.env.CLIENT_ID
-  const CLIENT_SECRET = process.env.CLIENT_SECRET
-  const REDIRECT_URIS = process.env.REDIRECT_URIS
-  const oAuth2Client = new google.auth.OAuth2(
-    CLIENT_ID, CLIENT_SECRET, REDIRECT_URIS[0])
+function authorize(credentials, receivers, subject, message, callback) {
+  const {client_secret, client_id, redirect_uris} = credentials.installed
```

Comment:
We switched from using environment variables to checking for the `credentials.json` file. Just wanted to make sure we understand the implications of this: 

By using file-based credentials, it is harder to deploy this application to a server. Since the `credentials.json` file is not checked into our codebase (which is good), everytime we deploy someone has to manually copy the file into the server. If we use envirnoment variables, they would be persisted between multiple re-deploys on a server.

It's not a major issue if we don't have automated deployments, but I just wanted to make sure this consequence was pointed out. Let me know if it's not worth to fix this right now.

Reply:
unless we automate deploys in the next few days I feel like this is easier and I felt like updating the env variables when the token updated was kinda wrong

Reply:
I don't think it'll be a big issue considering we are essentially done

Reply:
Ok, makes sense!